widgetssdk/src/main/java/com/glia/widgets/GliaWidgets.java

```diff
@@ -323,6 +328,22 @@ public static String getWidgetsCoreSdkVersion() {
         return BuildConfig.GLIA_CORE_SDK_VERSION;
     }
 
+    /**
```

Comment:
Maybe it is better not to provide the public doc for the function.

Reply:
Makes sense. Removed the doc completely, taking into account that Widgets are in a public repository. fdf5bc04bb9466b9c9e71a7e1d78520486d3e818